src/matchers/toBeArrayOfSize/index.test.js

```diff
@@ -0,0 +1,47 @@
+import each from 'jest-each';
+
+import matcher from './';
+
+expect.extend(matcher);
+
+describe('.toBeArrayOfSize', () => {
+  {
+    const size = 1;
+    test(`passes when given an array of size 1, where size === ${size}`, () => {
+      expect([1]).toBeArrayOfSize(size);
+    });
+  }
+  {
+    const size = 134;
```

Comment:
What do you think to these scoped variables? They have been use to avoid writing errors.

Reply:
I'm not torn either way 😃